Champions/MasterYi/R.cs

```diff
@@ -0,0 +1,49 @@
+using LeagueSandbox.GameServer.API;
```

Comment:
What's going on with the indentation in this file? Half seem to use tabs and the other half spaces, please make them consistent.